processor/workers.go

```diff
@@ -243,23 +280,48 @@ func blankState(
 	endComments [][]byte,
 	endString []byte,
 	langFeatures LanguageFeature,
-) (int, int64, []byte, [][]byte) {
+) (int, int64, []byte, [][]byte, bool) {
 	switch tokenType, offsetJump, endString := langFeatures.Tokens.Match(fileJob.Content[index:]); tokenType {
 	case TMlcomment:
 		if langFeatures.Nested || len(endComments) == 0 {
 			endComments = append(endComments, endString)
 			currentState = SMulticomment
 			index += offsetJump - 1
-			return index, currentState, endString, endComments
+			return index, currentState, endString, endComments, false
 		}
 
 	case TSlcomment:
 		currentState = SComment
-		return index, currentState, endString, endComments
+		return index, currentState, endString, endComments, false
 
 	case TString:
+		// TODO But was it a special state which means we should ignore \ further on?
```

Comment:
Looks like this block was copy/pasted from `stringState` above. Can we lift the duplication out to a function?

Reply:
Thanks for reminding me! This is exactly why I wanted someone to review this.

Reply:
Done.